rich/progress.py

```diff
@@ -613,6 +608,15 @@ def __init__(
         self.print = self.console.print
         self.log = self.console.log
 
+    @classmethod
+    def default_columns(cls) -> Tuple[ProgressColumn, ...]:
+        return (
```

Comment:
Could you change this to `get_default_changes`  and add a brief docstring...

Reply:
Did you mean get_default_columns?

Reply:
Yeah, `get_default_columns`.  Another case of the Mondays...

Reply:
I've got a decent docstring, but just want to double-check on the name. Do you *really* want get_default_columns()? I usually associate getters with instances, not the class. For this method, I'm not sure what value the "get_" prefix adds.

Reply:
Leaning towards `get`. Implies it does something more than expose data.

Reply:
See updates